packages/block-library/src/block/block.json

```diff
@@ -10,6 +10,12 @@
 	"attributes": {
 		"ref": {
 			"type": "number"
+		},
+		"layout": {
```

Comment:
If I'm not mistaken, I think you might not need to explicitly add the layout attribute in order to set a default? Within the `layout` object in `supports`, you can set a `default` object. For example, here's the one from the Columns block: https://github.com/WordPress/gutenberg/blob/ad92865dac36d1701c6d14e4f81f947d7c3f9907/packages/block-library/src/columns/block.json#L48-L56

Reply:
That seems to work - and saves having a documented attrib for the block that shouldn't actually be used. Thanks.